Dockerfile

```diff
@@ -0,0 +1,6 @@
+# see https://pptr.dev/guides/docker and https://github.com/puppeteer/puppeteer/pkgs/container/puppeteer
+FROM ghcr.io/puppeteer/puppeteer:23
```

Comment:
I assume 23 is the version. Why don't we use the latest version? That is implicit in the installation instructions, though I have to admit fixing the version might prevent the code from breaking. But OTOH one needs to remember to update this when a new version is released, which IMHO is the more subtle error. So knowing next to nothing about Docker :-), WDYT about using "HEAD" here?